apps/fulfillment-e2e/src/integration/customer-note.cy.ts

```diff
@@ -0,0 +1,23 @@
+import { CustomerNoteModalFragment } from '../support/page_fragments/customer-note-modal.fragment';
+import { PickingListsFragment } from '../support/page_fragments/picking-lists.fragment';
+
+const pickingListsFragment = new PickingListsFragment();
+const customerNoteFragment = new CustomerNoteModalFragment();
+
+describe('Customer note', () => {
```

Comment:
It is better to change the title here to "When the user opens customer note" to give move sense of that is going on here

Reply:
I will change it in the next PR